docs/source/onnx.rst

```diff
@@ -332,18 +342,18 @@ The process for adding a symbolic function depends on the type of operator.
 ATen operators
 ^^^^^^^^^^^^^^
 
-`ATen <https://pytorch.org/cppdocs/#aten>`_ is PyTorch’s built-in tensor library.
+`ATen <https://pytorch.org/cppdocs/#aten>`_ is PyTorch's built-in tensor library.
 If the operator is an ATen operator (shows up in the TorchScript graph with the prefix
 ``aten::``), make sure it is not supported already.
 
 List of supported operators
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Visit the auto generated :doc:`list of supported ATen operators <../onnx_supported_aten_ops>`
+Visit the auto generated :doc:`list of supported TorchScript operators <../onnx_supported_aten_ops>`
```

Comment:
Is TorchScript operator actually an official name than ATen?

Reply:
Not sure if it is an official name, but we have quantized and other operators that are not ATen